docker-compose.yml

```diff
@@ -7,14 +7,26 @@ services:
     volumes:
       - type: bind
         source: '${DATA_DIR}/${REDIS_DATA_DIR}'
-        target: '/data'
+        target: /data
     networks:
       - default
 
   rq_worker:
     image: 'd3m-experimenter:latest'
     env_file:
       - ./.env
+    volumes:
+      - type: bind
+        source: '${DATASETS_DIR}'
+        target: /datasets
+        read_only: true
+      - type: bind
+        source: '${DATA_DIR}'
+        target: /data
+      - type: bind
+        source: '${EXPERIMENTER_DIR}'
+        target: /d3m-experimenter
+        read_only: true
```

Comment:
Why did you need to mount the experimenter code as a volume?

Reply:
I am actually not sure if the last mount is necessary. The other two are needed to show the worker where to get the data and put the data. I will run some tests with and without the /d3m-experimenter volume mount before resolving this.